### PR TITLE
terms and privacy

### PR DESCRIPTION
**Pull request by gerouvi**
New terms and privacy

refs #189

### CODE REVIEW

File: src/themes/default/components/Footer.tsx [hunk 1]
```diff
@@ -107,8 +107,8 @@ const Footer = () => {
           <Trans
             i18nKey='footer.terms_and_privacy'
             components={{
-              link1: <Link href='https://aragon.org/terms-and-conditions' target='_blank' color='gray' />,
-              link2: <Link href='https://aragon.org/privacy-policy' target='_blank' color='gray' />,
+              link1: <Link href='/terms' color='gray' />,
+              link2: <Link href='/privacy' color='gray' />,
```
Comment (elboletaire) [inline src/themes/default/components/Footer.tsx:111]: Should be a router link and use `to`.

File: src/components/TermsAndPrivacy/Terms.tsx [hunk 2]
```diff
@@ -0,0 +1,182 @@
+import { Flex, Heading, ListItem, Text, UnorderedList } from '@chakra-ui/react'
+
+const Terms = () => {
+  return (
```
Comment (elboletaire) [inline src/components/TermsAndPrivacy/Terms.tsx:4]: Friendly reminder you can simplify this to:

```tsx
const Terms = () => (
)
```

File: src/components/TermsAndPrivacy/Privacy.tsx [hunk 3]
```diff
@@ -0,0 +1,119 @@
+import { Box, Flex, Heading, Link, List, ListItem, Text } from '@chakra-ui/react'
+
+const Privacy = () => {
+  return (
```
Comment (elboletaire) [inline src/components/TermsAndPrivacy/Privacy.tsx:4]: Same as in terms. No need to use `{ return` when there's no logic in the component.

**Review by elboletaire — CHANGES_REQUESTED** [review]

**Comment by elboletaire** [comment]
@jpaulet I do now realize the onvote theme does not have the terms and conditions links visible until you login (?) 😕

**Comment by gerouvi** [comment]
> @jpaulet I do now realize the onvote theme does not have the terms and conditions links visible until you login (?) 😕

We could also put it in the footer, right?

**Comment by jpaulet** [comment]
Yes, let's add it to the footer in the Onvote app.

**Review by elboletaire — APPROVED** [review]

**Comment by elboletaire** [comment]
Aren't we translating these pages?

**Pull request merged**